### DISCUSSION TITLE
Can I use deepl features like "Create your own glossary" and "Select alternative words or phrases"

### DISCUSSION THREAD

**Discussion by nikolaj-kaplan** [Fekide/strapi-plugin-translate#325] [Q&A]
It seems that Deepl has some features implemented in their various UIs. Are these features supported in this plugin? Or do you have any plans to support them?

**Comment by sargreal** [comment]
As you can see in #28, glossaries are a planned feature. However there is no time frame on that.

The "Select alternative words or phrases" feature is currently not planned as of the complexity and the DeepL API not having these features directly available. However if you really want that feature you can always open a PR
